weave-js/src/components/Timestamp.tsx

```diff
@@ -38,6 +38,18 @@ const formatSmallTime = (then: moment.Moment): string | null => {
   const hours = Math.floor(diffMs / (1000 * 60 * 60));
   const days = Math.floor(diffMs / (1000 * 60 * 60 * 24));
 
+  const years = now.diff(then, 'years');
+  if (years > 0) {
+    const remainingDays = days - years * 365;
+    if (remainingDays < 7) {
+      if (years === 1) {
+        return '1y';
+      } else if (years > 1) {
+        return `${years}y`;
```

Comment:
Doesn't this return work for the years === 1 case too?

Reply:
... yes